src/todoFunctions.js

```diff
@@ -0,0 +1,35 @@
+export function addTask(tasks, description) {
+  const newIndex = tasks.length;
+  const newTask = {
+    description,
+    completed: false,
+    index: newIndex + 1,
+  };
+  tasks.push(newTask);
+}
+
+function updateIndexes(tasks) {
+  tasks.forEach((task, index) => {
+    task.index = index + 1;
+  });
+}
+
+export function deleteTask(tasks, index) {
+  tasks.splice(index, 1);
+  updateIndexes(tasks);
+}
+
+export function editTaskDescription(tasks, index, newDescription) {
+  tasks[index].description = newDescription;
```

Comment:
- [ ] I'm unable to edit/update tasks from the UI. Kindly check and fix accordingly.

**Unable to edit entry 'one':**
[screencast-localhost_8080-2023.06.15-04_21_17.webm](https://github.com/bohaz/To-do-list/assets/12745474/32346bf2-df7d-4e29-9bcf-4f224d273f54)